public-html/aboutus.php

```diff
@@ -41,6 +61,10 @@
         </div>
     </div>
 
+    <!-- Include scripts -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" charset="utf-8"></script>
+    <script src='<?php echo $menu->paths['js'] . "/main.js" ?>'></script>
+
```

Comment:
These lines are repeated everywhere, so maybe you can put them in some function/ code block  and call them everywhere.

Reply:
Need to create a template render class for it.

Reply:
YES, am open to help with that. THAT will more cool